homeassistant/components/modbus/climate.py

```diff
@@ -276,7 +276,7 @@ def _read_register(self, register_type, register) -> float | None:
     def _write_register(self, register, value):
         """Write holding register using the Modbus hub slave."""
         try:
-            self._hub.write_registers(self._slave, register, [value, 0])
+            self._hub.write_registers(self._slave, register, value)
```

Comment:
this change looks Ok,  it would  be nice to see a test case as well.

@vzahradnik you wrote this code, was there a reason for the 0 ?

Reply:
I don't remember what was the reason. Let's keep it as it's proposed here.